syntax = "proto3";

service RecognitionFacial {
  rpc UploadTraining(stream Chunk) returns (Reply) {}
  rpc UploadRecognition(stream Chunk) returns (Reply) {}
}

message Chunk {
  bytes buffer = 1;
  string nameFile = 2;
  string identifier = 3;
  string ipPort = 4;
  int32 qtde = 5;
}

message Request {
  string name = 1;
}

message Reply {
  string message = 1;
  int32 length = 3;
}
